hip-service/Link/Patient/PatientVerification.cs

```diff
@@ -1,27 +1,86 @@
-#nullable enable
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
 using System.Threading.Tasks;
 using hip_service.OTP;
 using HipLibrary.Patient.Model.Response;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using Optional;
 
 namespace hip_service.Link.Patient
 {
     public class PatientVerification: IPatientVerification
     {
-        private readonly OtpVerification otpVerification;
-
-        public PatientVerification(OtpVerification otpVerification)
+        private readonly IConfiguration configuration;
+        private HttpClient httpClient;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field 'httpClient' and declare it as a local variable in the relevant methods.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4807294)